.travis.yml

```diff
@@ -21,14 +21,20 @@ env:
   matrix:
     # we recommend new addons test the current and previous LTS
     # as well as latest stable release (bonus points to beta/canary)
-    - stage: "Additional Tests"
-      env: EMBER_TRY_SCENARIO=ember-lts-2.18
-    - env: EMBER_TRY_SCENARIO=ember-lts-3.4
-    - env: EMBER_TRY_SCENARIO=ember-lts-3.8
-    - env: EMBER_TRY_SCENARIO=ember-release
-    - env: EMBER_TRY_SCENARIO=ember-beta
+    - EMBER_TRY_SCENARIO=ember-lts-2.12
```

Comment:
I think the config/ember-try.js doesn't include 2.12 or 2.16 scenarios

Reply:
fixed, thank you @rwjblue !